src/components/Api.js

```diff
@@ -0,0 +1,105 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    (this._baseUrl = baseUrl), (this._headers = headers);
```

Comment:
Needs correction: please, don't split statements with a comma. Every statement should have own line:

```js
this._baseUrl = baseUrl;
this._headers = headers;
```